sklearn/linear_model/_perceptron.py

```diff
@@ -148,13 +153,14 @@ class Perceptron(BaseSGDClassifier):
     https://en.wikipedia.org/wiki/Perceptron and references therein.
     """
     @_deprecate_positional_args
-    def __init__(self, *, penalty=None, alpha=0.0001, fit_intercept=True,
+    def __init__(self, *, penalty=None, alpha=0.0001, l1_ratio=0.15,
```

Comment:
Unfortunately introducing `l1_ratio=.15` might produce changes in the results of users who had set `penalty='elasticnet'`, since it used to be hardcoded to 0 before.

We can:

- set the default to 0, but that would be slightly weird since just setting `penalty='elasticnet'` would still be using L2. It would also be inconsistent with SGDClassifier.
- set the default to 'warn' which is equivalent to 0, but indicates that the default will change from 0 to .15 in version 0.26
- not care about all this and consider that this is a bugfix. In this case we can just set the default as it is: 0.15


I have a slight preference for option 3 since it's the simplest and it's likely that this won't affect many people.

Thoughts @rth @glemaitre @ogrisel @thomasjpfan?

Reply:
I'm also in favor of option 3 as tend to perceive it as a bugfix.

Reply:
And why 0.15 as default, not, for instance, 0.5?

Reply:
I also slightly prefer option 3 and consider this a bug fix.

Reply:
Agreed, seems like a bugfix. And the default should be consistent with SGDClassifier, which has 0.15, probably for no real reason.